ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/AttestationUtil.java

```diff
@@ -161,43 +196,58 @@ public static UnsignedLong get_current_epoch_boundary_attesting_balance(BeaconSt
   public static UnsignedLong get_previous_epoch_boundary_attesting_balance(BeaconState state)
       throws Exception {
 
+    List<Integer> previous_epoch_boundary_attester_indices =
+        get_previous_epoch_boundary_attester_indices(state);
+
+    return AttestationUtil.get_total_attesting_balance(
```

Comment:
You don't need to prefix the call to `get_total_attesting_balance ` with `AttestationUtil` since this code is in `AttestationUtil`.